_data/sites.json

```diff
@@ -3788,7 +3788,7 @@
         "name": "CreditExpert",
         "url": "https://www.creditexpert.co.uk",
         "difficulty": "hard",
-        "notes": "You can cancel by email but only if your membership includes insurance. Otherwise, an email cancellation will be ignored. You have to call 0800 561 0083 to cancel your account. This is the only way. More info here: <a href=\"http://experian.metafaq.com/help/CreditExpertBRS/Cancel_and_duration/CancelBRS\">http://experian.metafaq.com/help/CreditExpertBRS/Cancel_and_duration/CancelBRS</a>",
+        "notes": "You can cancel by email but only if your membership includes insurance. Otherwise, an email cancellation will be ignored. You have to call 0800 561 0083 to cancel your account. This is the only way. More info here: [http://experian.metafaq.com/help/CreditExpertBRS/Cancel_and_duration/CancelBRS](http://experian.metafaq.com/help/CreditExpertBRS/Cancel_and_duration/CancelBRS)",
```

Comment:
```suggestion
        "notes": "You can cancel by email but only if your membership includes insurance. Otherwise, an email cancellation will be ignored. You have to call 0800 561 0083 to cancel your account. This is the only way. More info [here](http://experian.metafaq.com/help/CreditExpertBRS/Cancel_and_duration/CancelBRS)",
```